sqldelight-gradle-plugin/src/test/fixtures/keyword_identifiers/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test (
+  "DESC" TEXT,
```

Comment:
(because if it's the latter, we should be stripping the quote for the generated field and this example should fail with duplicate names)